package.json

```diff
@@ -0,0 +1,18 @@
+
+
+
+
+{
+  "name": "s53_aniview",
+  "version": "1.0.0",
+  "description": "As anime is really popular these days, everyone has their own preferences and taste for their choice to watch. So, for everyone, they have their own underrated anime preference.",
+  "main": "index.js",
+  "scripts": {
+    "start": "node index.js", 
```

Comment:
Consider adding a 'dev' script for development purposes. This script could run the application in a development mode with hot-reloading enabled. For example, you could use nodemon for this purpose. [medium]